packages/s3-store/index.ts

```diff
@@ -30,6 +31,7 @@ type MetadataValue = {
   file: Upload
   'upload-id': string
   'tus-version': string
+  'created-at': string
```

Comment:
Maybe I'm misunderstanding something but `file` is `Upload` which already has `creation_date`? Can't we reuse that?